src/AnalyticsConfig.ts

```diff
@@ -7,14 +7,10 @@ import { Region } from './Region';
  * @public
  */
 export interface AnalyticsConfig {
-  /** The API Key or OAuth for accessing the Analytics Events API
-   * Only one of key or bearer should be set.
-   * */
-  key?: string;
-  /** The bearer token for accessing the Analytics Events API.
-   * Only one of key or bearer should be set.
-   * */
-  bearer?: string;
+  /** Used for specifying if an API Key or Bearer Token is used for the authorization property. */
+  authorizationType: 'apiKey' | 'Bearer';
```

Comment:
I know I put this as the example, but do you want the two types here to have a consistent capitalization?